webapp/cube/views.py

```diff
@@ -45,6 +54,8 @@
     edx_session,
 )
 
+ua_contracts_session = Session()
```

Comment:
add `talisker.requests.configure(ua_contracts_session)`

Reply:
Can I do instead `talisker.requests.get_session()`

Reply:
No, that will just end up reusing and existing session. You need to configure the created session.

Reply:
Done